website/src/pages/index.module.scss

```diff
@@ -0,0 +1,159 @@
+@use '../css/_fonts.scss' as *;
+
+.heroBanner {
+  --ifm-heading-color: #fff;
+
+  width: 100%;
+  height: calc(357px + 23vw);
+  padding-top: 20px;
+  min-height: 450px;
+  max-height: 725px;
+  overflow: hidden;
+  position: relative;
+  background: linear-gradient(150deg, #000000, #15216b, #ffffff);
+  background-size: 300% 300%;
+  animation: gradientShift 4s infinite;
+
+  @media only screen and (max-width: 550px) {
+    height: calc(450px + 10vw); /* Adjust height */
+    min-height: 550px;
+  }
+
+  @media only screen and (max-width: 445px) {
+    height: calc(500px + 10vw); /* Further increase height */
+    min-height: 600px; /* Ensure extra space */
+  }
+
+  @keyframes gradientShift {
+    0% {
+      background-position: 0% 50%;
+    }
+    25% {
+      background-position: 50% 100%;
+    }
+    50% {
+      background-position: 100% 50%;
+    }
+    100% {
+      background-position: 50% 0%;
+    }
+  }
+
+  @media only screen and (max-width: 375px) {
+        background-position-x: -379px;
+        background-position-y: -53px;
+        background-size: 1100px;
+    }
+
+    @media only screen and (max-width: 890px) {
+        padding-top: 32px;
+        padding-bottom: 32px;
+    }
+
+    @media only screen and (max-width: 1200px) {
+        --ifm-spacing-horizontal: 1.625rem;
+    }
+
+    @media only screen and (min-width: 1600px) {
+        background-position-x: center;
+        background-position-y: bottom;
+        background-size: cover;
+    }
+}
+
+.title {
+    @include h2;
+    font-size: 120px;
+    line-height: normal;
+    letter-spacing: -6px;
+    margin-left: -10px;
+
+    @media only screen and (max-width: 890px) {
+        font-size: 72px;
+        line-height: normal;
+        letter-spacing: -3.6px;
+        margin-left: -6px;
+    }
+}
+
+.tagline {
+    @include t1;
+    max-width: 450px;
+    min-height: 60px;
+    margin-top: -30px;
+    margin-bottom: 32px;
+    letter-spacing: 0.1px;
+
+    @media only screen and (max-width: 890px) {
+        margin-top: -22px;
+        margin-bottom: 21px;
+    }
+}
+
+.buttons {
+    height: 50px;
+}
+
+.linkText {
+    @include t2;
+    font-size: 17px;
+    color: var(--ifm-color-gray-900);
+    background-color: var(--ifm-button-background-color);
+    min-width: 250px;
+    padding: 12px 20px;
+    border: 0;
+    border-radius: 10px;
+}
+
+.bottomRightGif {
+  position: absolute;
+  bottom: 20px;
+  right: 20px;
+  width: 850px;
+  height: auto;
+  border-radius: 8px;
+
+  @media only screen and (max-width: 1200px) {
```

Comment:
Look, the bottomRightGif CSS is not good. Even 7+ media queries with hard-coded values don't save your day:

![image](https://github.com/user-attachments/assets/17236d6c-a96c-4879-8eff-e27268fd8fc0)

There are two approaches to fix this:

1. If you stick to absolute positioning, use linear interpolation (lerp) principles. Instead of discrete breakpoints, we need a continuous function that smoothly transitions between viewport sizes:
   ```css
   .bottomRightGif {
     position: absolute;
     /* Linear scaling function: f(x) = mx + b where:
        m = (y2-y1)/(x2-x1) for the scaling rate
        b = y-intercept for minimum values */
     width: min(850px, calc(40vw + 300px));
     right: max(20px, calc(2vw + 10px));
     bottom: max(20px, calc(2vw + 10px));
   }
   ```
   This follows the principle of linear regression where we map viewport width to element dimensions (dependent variable), creating a continuous scaling function rather than discrete steps.
   The numbers in the example are random, magical and serve just for illustration. The syntax about `min`/`max` has to be taken with a grain of salt — it should [work in CSS 2020+](https://developer.mozilla.org/en-US/docs/Web/CSS/min#browser_compatibility), but I never checked it personally as I touch website code not that often, sorry.

2. Maybe consider CSS Grid, who knows?
   ```css
   .heroBanner {
     display: grid;
     grid-template-columns: 1fr min(850px, 100%); // again, numbers just for illustration
   }
   ```

The current implementation with 7+ media queries is essentially trying to approximate a linear function with step functions - that's why you get those awkward jumps between breakpoints.
Either implement proper linear scaling or use some layout systems.

P.S. Whatever approach you choose, test it with various screens (360px, and drag width slider slowly and check). The current implementation breaks on 1200px, for example. We don't want that in production.